### ISSUE TITLE
Peer review for hw07-rgryba

### ISSUE THREAD

**Issue by ziqiangt** [STAT545-UBC-students/hw07-rgryba#1]
|    **Topic**    | **Excellent** |**Satisfactory** |**Needs Work** |
|----------------|---------------|------------------|------------------|
| Coding style  |    :thumbsup:    |                     |                           |
| Coding strategy      |    |           :thumbsup:               |                           |
| Presentation: graphs ||                          |                😟            |
| Presentation: tables |  |          :thumbsup:              |                          |
| Achievement, activity |     |                       |           😟                 |
| Ease of access |    |         :thumbsup:             |                       |

Remarks:
rgryba did a good job. Here are some comments that might interest you. 
1. Your example "an invalid location" works fine since it returns 0 row .
2. Your "Vignette code" works fine while it might need more in line comments and chunk comments.
3. I did not see R scrips in the R folder
3. I did not see any testthat case.
3. Good readme file and useful navigation hyper-link.
In conclusion, you might refer to [Hw 07](http://stat545.com/Classroom/assignments/hw07/hw07.html) to view the requirements.
